src/components/vm/disks/diskEdit.jsx

```diff
@@ -78,8 +78,8 @@ const CacheRow = ({ onValueChanged, dialogValues, idPrefix, shutoff }) => {
     );
 };
 
-const BusRow = ({ onValueChanged, dialogValues, idPrefix, shutoff }) => {
-    const busTypes = ['sata', 'scsi', 'usb', 'virtio'].map(type => ({ value: type }));
+const BusRow = ({ onValueChanged, dialogValues, diskDevice, idPrefix, shutoff }) => {
+    const busTypes = diskBusTypes[diskDevice].map(type => ({ value: type }));
```

Comment:
This assumes that `diskDevice` only ever has the values `disk` or `cdrom`. Is that safe? Could it have any other value or be undefined? Then this would crash.

Reply:
Indeed. Added the missing device types and disabled the button just for the case a type is something that's not documented.